docker/compose/suzuka-full-node/docker-compose.ledger-should-progress.yml

```diff
@@ -0,0 +1,70 @@
+services:
+
+  # This service checks if the ledger version has increased every LEDGER_INCREASE_PERIOD_SECONDS
+  ledger-should-progress:
+    image: alpine:latest
+    container_name: ledger-should-progress
+    command: >
+      sh -c '
+        # install jq and curl for busybox
+        apk add --no-cache jq curl
```

Comment:
There may be a fatter base image with these utilities installed, but OK.

Reply:
#717